src/presentation/public/service-worker.js

```diff
@@ -0,0 +1,70 @@
+// Event listener for when the service worker is installed
+self.addEventListener('install', (event) => {
```

Comment:
There are some linting errors, run `npm run lint:eslint` locally and these should be fixed.